### PR TITLE
feat: Translate section `Linux a nevidomí uživatelé`

### PR DESCRIPTION
**Pull request by m-czernek**
This PR fixes issue #10 .

### CODE REVIEW

File: docs/handout_en.md [hunk 1]
```diff
@@ -24,30 +24,30 @@ myslíme tím, že používáme linuxovou distribuci. Linuxová distribuce je
...
 
-## Linux a nevidomí uživatelé
+## Linux and visually impaired users
 
-Protože tento workshop byl vytvořen pro začátečníky, předpokládám, že
-budete Linux používat skrze jeho grafické rozhraní. Jen pro úplnost
-dodávám, že jej lze ovládat i skrze čistě textové rozhraní.
+This workshop is aimed at beginner users, so I expect most of you
+will use Linux with the graphical user interface. However, note
+that you can fully control Linux by only using a text interface as well.
```
Comment (vojtapolasek) [inline docs/handout_en.md:31]:
```suggestion
that you can fully control Linux by using only a text interface as well.
```

File: docs/handout_en.md [hunk 2]
```diff
@@ -24,30 +24,30 @@ myslíme tím, že používáme linuxovou distribuci. Linuxová distribuce je
...
+[Orca](https://help.gnome.org/users/orca/stable/) is the only
+screen reader that works completely on Linux. You can use other
+screen readers on Linux, but only Orca works with the graphical interface.
+Other screen readers work only in the command line interface, which
+is not in scope of this workshop.
 
-Budeme používat distribuci Fegora. Název Fegora vznikl spojením slov
-Fedora a Agora. [Fedora](https://mojefedora.cz/) je Linuxová distribuce
-podporovaná firmou [Red Hat](https://redhat.com/) a
+The subject of this workshop is Fegora. The name Fegora a contraction
```
Comment (vojtapolasek) [inline docs/handout_en.md:39]:
```suggestion
The subject of this workshop is Fegora. The name Fegora a conjunction
```

File: docs/handout_en.md [hunk 3]
```diff
@@ -24,30 +24,30 @@ myslíme tím, že používáme linuxovou distribuci. Linuxová distribuce je
...
 [Agora](https://www3.teiresias.muni.cz/cz/vzdelavani-sport-a-volny-cas/agora)
-je cyklus workshopů o informačních technologiích pro zrakově postižené.
-Konkrétně jsme vycházeli z odnože [Fedora Mate
-Compiz](https://spins.fedoraproject.org/mate-compiz/). Tato
-verze sama o sobě byla relativně dobře přístupná pro nevidomé uživatele,
-ale provedli jsme různá vylepšení, aby se používala ještě lépe.
-Především jsme zajistili, aby odečítač mluvil už na přihlašovací
-obrazovce a nainstalovali některé zajímavé programy, například pro
-rozpoznávání textu pomocí technologie OCR.
+is a Czech series of workshop about information technologies aimed at visually
```
Comment (vojtapolasek) [inline docs/handout_en.md:43]:
```suggestion
is a Czech series of workshops about information technologies aimed at visually
```

File: docs/handout_en.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ virtuální počítač a Linux by měl automaticky naběhnout.
 Ještě jednou vám gratuluji, právě jste prošli tím nejtěžším, co
 začátečníky s Linuxem čeká a občas také odradí. Teď už půjde všechno
 relativně hladce. Jak jsem psal výše, po úspěšném startu systému
-uslyšíte hlášku "Čtečka obrazovky zapnuta". 
+uslyšíte hlášku "Čtečka obrazovky zapnuta".
```
Comment (vojtapolasek) [inline docs/handout_en.md:141]: What for?

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you very much for the translaiton. Please see comments.

File: docs/handout_en.md (hunk 2 above)
Comment (m-czernek) [reply docs/handout_en.md:39]: I used `contraction` based on [this](https://en.wikipedia.org/wiki/Smog) article (that uses contraction). I think `blend` is the best; WDYT? 

What we want to say is `portmanteau` but IMHO many readers won't know what that means.

```suggestion
The subject of this workshop is Fegora. The name Fegora a blend
```

File: docs/handout_en.md (hunk 4 above)
Comment (m-czernek) [reply docs/handout_en.md:141]: See previous PR comment

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
Thank you for the changes. Merging.

**Pull request merged**